plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/backstageRequest.ts

```diff
@@ -127,7 +127,12 @@ export function createHttpBackstageAction(options: { config: Config }) {
         body: inputBody,
       };
 
-      if (token) {
+      const authToken = getObjFieldCaseInsensitively(
```

Comment:
Could you add a test for this change to the backstageRequest.test.ts file?

Reply:
Done